test/e2e/sagemaker/replacement_values.py

```diff
@@ -71,5 +71,6 @@
     "SAGEMAKER_DATA_BUCKET": get_bootstrap_resources().DataBucketName,
     "XGBOOST_IMAGE_URI": f"{XGBOOST_IMAGE_URIS[get_aws_region()]}/sagemaker-xgboost:1.0-1-cpu-py3",
     "PYTORCH_TRAIN_IMAGE_URI": f"{PYTORCH_TRAIN_IMAGE_URIS[get_aws_region()]}/pytorch-training:1.5.0-cpu-py36-ubuntu16.04",
-    "SAGEMAKER_EXECUTION_ROLE_ARN": get_bootstrap_resources().ExecutionRoleARN
+    "SAGEMAKER_EXECUTION_ROLE_ARN": get_bootstrap_resources().ExecutionRoleARN,
+    "TRANSFORM_MODEL": f"s3://{get_bootstrap_resources().DataBucketName}/sagemaker/batch/model.tar.gz"
```

Comment:
For our other CRDs we hardcode the path values into the YAML files. I understand this model is not created from a YAML (you're calling boto3 directly), but I feel for consistency we should not have references to data paths in this file.

Reply:
I moved it to the test file